map_data/geographical_region.txt

```diff
@@ -794,9 +815,16 @@ custom_eastern_kingdoms_jungles = {
 }
 custom_jungles_except_pandaria = {
 	regions = {
-		world_northrend_sholazar_basin
 		custom_eastern_kingdoms_jungles
 		custom_south_seas_except_deserts
+		world_kalimdor_feralas
+		world_northrend_sholazar_basin
+	}
+	duchies = {
+		d_echo_isles
+		d_seething_shore
+		d_glaive_isle
+       		d_lost_isles
 	}
```

Comment:
These duchies are included in `custom_south_seas_except_deserts`

Reply:
These are duchies of 
world_kalimdor_durotar (echo isles)
world_kalimdor_veiled_isles (seething shore)
world_south_seas_kezan (glaive isle/lost isles)

![image](https://user-images.githubusercontent.com/74790101/115279664-fd898c80-a114-11eb-87f6-de9ca7145315.png)

It is not included, or have I misunderstood?

Reply:
> It is not included, or have I misunderstood what you were saying?

No, my bad. Though I moved the Lost Isles and Glaive Isle in a36d7709a8862d3f3ec124aa8bb5288da5ee22b2.